Updated code per design file

AceIsHuman opened the pull request:
Created UI based on design file and added a fixed navigation bar.

AceIsHuman committed:
Created UI based on design file

AceIsHuman committed:
Updated Nav Bar

RyanNFreeman commented:
Looks great. Maybe a couple stylistic recommendations, like removing the underlines on the anchor tags. I'd also style your button with `cursor: pointer;`

AceIsHuman committed:
added newFile.txt to update brach

AceIsHuman committed:
deleted newFile.txt and added name to README.md

AsaOfDiamonds committed:
Merge pull request #1 from AceIsHuman/aciel-ochoa

Added name to README.md

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 515.07kb -> 467.04kb (9.32%)

/great-idea-website/img/logo.png -- 3.26kb -> 2.05kb (37.11%)
/great-idea-website/img/header-img.png -- 187.27kb -> 164.10kb (12.37%)
/design-files/desktop.jpg -- 302.21kb -> 279.07kb (7.66%)
/great-idea-website/img/mid-page-accent.jpg -- 22.33kb -> 21.83kb (2.26%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

AceIsHuman committed:
Merge pull request #2 from AceIsHuman/imgbot

[ImgBot] Optimize images
